clientgui/AsyncRPC.cpp

```diff
@@ -149,7 +149,9 @@ void *RPCThread::Entry() {
     {
         // On Mac / Unix / Linux, set "C" locale for this thread only
         RPC_Thread_Locale = newlocale(LC_ALL_MASK, "C", NULL);
+#if !defined(__NetBSD__)
```

Comment:
This and the same below should be unnecessary. It should be handled by the #ifndef NO_PER_THREAD_LOCALE test above.